static/css/learnocaml_student_view.css

```diff
@@ -366,6 +378,9 @@ tr.exercise-highlight {
 }
 
 /* -------------------- ACE overriding ---------------------------- */
+/* XXX Erik: this section looks like dead CSS code,
```

Comment:
A good practice would be to add an issue about this problem and to refer to this issue in the code so that we are sure it is properly tracked down.

Reply:
Good point; I will do this.